apps/lend/src/utils/utilsStorage.ts

```diff
@@ -36,9 +36,11 @@ function getTheme(svThemeType: string | undefined) {
   }
 }
 
-function getWalletName(walletName: string | undefined, timestamp: string | undefined) {
-  const isStaled = walletName && timestamp && dayjs().diff(+timestamp, 'days') > 5
-  return isStaled || !walletName ? '' : walletName
+function getWalletName(timestamp: string | undefined) {
+  const storedValue = window.localStorage.getItem('onboard.js:last_connected_wallet')
```

Comment:
I don't think it's the best idea to use a hardcoded access key. Try to find a way to get the value of this field using onboard library

Reply:
Actually, I don't need this, the `autoConnectLastWallet: true` handles this situation now. I will remove.